src/app/shared/components/List/index.tsx

```diff
@@ -242,7 +243,7 @@ ListProps<T, F>): JSX.Element {
       </div>
 
       {/* BODY */}
-      <div id="scrollableList" className="flex h-full flex-col overflow-y-auto" ref={ref}>
+      <div id="scrollableList" className="flex h-full flex-col overflow-y-auto overflow-x-hidden" ref={ref}>
```

Comment:
This works good for keeping header and rows sticky, but raises another bug. When the user right-clicks an item and then focus out, it focuses the three dot menu, moving the horizontal scroll too. User can't get scroll back to normal.

https://github.com/internxt/drive-web/assets/34165099/be61cb75-f863-4c13-91c6-a85a00bc3b1e